### ISSUE TITLE
Why MSE loss always increase during training?

### ISSUE THREAD

**Issue by haoyayuzzz** [voxelmorph/voxelmorph#376]
**Task** (what are you trying to do/register?)

[please describe task here]

**What have you tried**

Please describe specifics of your approach // use of `vxm`

**Details of experiments**

Please carefully specify details about your experiments. If you are training, when what is the setup? What loss are you using? What does the convergence look like? If you are registering, please show example inputs and outputs. etc.

**Comment by haoyayuzzz** [comment]
Simply, I wonder if you observed MSE loss increasing during training and how to solve that? Thanks a lot!

**Comment by adalca** [comment]
@haoyayuzzz not really, i'd recommend lowering the learning rate as a start? It may be taking large steps in the wrong direction

**Comment by haoyayuzzz** [comment]
> @haoyayuzzz not really, i'd recommend lowering the learning rate as a start? It may be taking large steps in the wrong direction

But the Dice value of validation set is increasing too,  so I'm confused...

**Comment by adalca** [comment]
The Hard Dice  or Dice loss (which wants to go down)? 
Anyway we'd need to know a lot more about your setup, etc.

**Comment by haoyayuzzz** [comment]
> The Hard Dice or Dice loss (which wants to go down)? Anyway we'd need to know a lot more about your setup, etc.

Actually, I don't use VoxelMorph. I observed the case in my own research. Because you are so good at medical image registration, so I creat this issue and want to get some advices. In my research, I use MSE loss to do multi-modality registration. And I observed dice value (which wants to go up) is increasing when MSE loss in also increasing or quivering.

**Comment by adalca** [comment]
well, we'd still need more details to really comment -- but of course in general the ideal would be to lower MSE&smoothness if you're doing registration, or just lower Dice if you can use the semi-supervised style approaches, so I would stick with that even though the Dice happens to be improving in your runs. Try getting better gradients (high batch, etc) or lower LR

I'd recommend not using MSE for multi-modal, though, if you're directly operating on the images...
